### PR TITLE
Feature/html template sa

### PR DESCRIPTION
**Pull request by StevieAmb**
## Branch title

- Feature/HTML-template-sa

## Changes made

- The HTML boilerplate, the font styles, and the script tags with out javascript files were added to the index.html file. 

## Anything Refactored

- Nothing as of yet.

## Anything needed

- Nothing as of yet.

## Contributors

* Stevie Ambroise
* Alex Fritz
* Devon Wigle

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,4 +8,6 @@
   </head>
   <body>
   </body>
+  <script type="text/javascript" src="./src/Activity.js"></script>
```
Comment (alexmfritz) [inline index.html:11]: Nice work linking the JS files, Stevie!

**Comment by devonwigle** [comment]
A nice boilerplate!

File: index.html [hunk 2]
```diff
@@ -2,6 +2,9 @@
 <html>
   <head>
     <link rel="stylesheet" href="./styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
Comment (alexmfritz) [inline index.html:6]: What does preconnect mean?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
```
Comment (alexmfritz) [inline index.html:1]: This looks like a great!

**Pull request merged**